CHANGELOG.rst

```diff
@@ -14,6 +14,22 @@ Changelog
   :class:`~cryptography.hazmat.primitives.hashes.HashContext` were moved from
   :mod:`~cryptography.hazmat.primitives.interfaces` to
   :mod:`~cryptography.hazmat.primitives.hashes`.
+* :class:`~cryptography.hazmat.primitives.ciphers.base.CipherContext`,
+  :class:`~cryptography.hazmat.primitives.ciphers.base.AEADCipherContext`,
+  :class:`~cryptography.hazmat.primitives.ciphers.base.AEADEncryptionContext`,
+  :class:`~cryptography.hazmat.primitives.ciphers.base.CipherAlgorithm`, and
+  :class:`~cryptography.hazmat.primitives.ciphers.base.BlockCipherAlgorithm`
+  were moved from :mod:`~cryptography.hazmat.primitives.interfaces` to
+  :mod:`~cryptography.hazmat.primitives.ciphers.base`.
```

Comment:
We don't ever document the `.base` portion of the path, they're always exported in `ciphers`